docs/core/diagnostics/app_is_leaking_memory_eventual_crash.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Debugging a memory leak - .NET Core
+description: A tutorial walk-through, debugging a memory leak in .NET Core.
+author: sdmaclea
+ms.author: stmaclea
+ms.date: 10/14/2019
+---
+# Debugging a memory leak
+
+**This article applies to: ✓** .NET Core 3.0 SDK and later versions
+
+In this scenario, the endpoint will slowly start leaking memory and eventually will result in an out of memory exception. To diagnose this scenario, we need several key pieces of diagnostics data.
```

Comment:
In which scenario? Would be good to state first what's going on.

But some basic grammar suggestions as well:

```suggestion
In this scenario, the endpoint slowly starts leaking memory and eventually will result in an out of memory exception. To diagnose this scenario, you need several key pieces of diagnostics data.
```